ref/general/authentication.adoc

```diff
@@ -34,18 +38,30 @@ Open Liberty supports the following options for SSO authentication:
 - link:/docs/ref/feature/#socialLogin-1.0.html[Social Media Login] +
 Users can authenticate to websites that are hosted on the Open Liberty server by logging in with their social media accounts. When the `socialLogin-1.0` feature is enabled, login with Facebook, Twitter, Google, Linkedin, and GitHub is automatically configured. You can also define your own configuration for any social media platform that is based on the OAuth 2.0 or OpenID Connect 1.0 standards.
 
+- link:/docs/ref/general/#/docs/concept/sso-config-json.html[JSON Web Tokens] +
+JSON Web Tokens (JWT) are used to propagate the user identity that is established by Single sign-on (SSO) between different microservice applications. By configuring SSO user authentication with JWT, you can manage the identity that is shared between applications.
+
 - link:/docs/ref/general/#/docs/concept/sso-config-ltpa.html[LTPA cookies] +
 Lightweight Third Party Authentication (LTPA) provides SSO configuration support to authenticate users when they are accessing application servers. With LTPA enabled, you can encrypt, digitally sign, and securely transmit authentication-related data between servers.
 
 - link:/docs/ref/feature/#openidConnectServer-1.0.html[OpenID Connect]  +
-OIDC is a popular internet SSO protocol that works well with cloud-native applications. This protocol enables a client application to request the identity of the user as an ID token in a standardized, REST-like manner. The client application, or Relying Party, requests information about a user from an authorization server, known as an OpenID Connect Provider. After users are authenticated by an OpenID Provider, they can access any Relying Party that is configured for that Provider, without having to manage unique accounts for each application or website.
+OIDC is a popular internet SSO protocol that works well with cloud-native applications. This protocol enables a client application to request the identity of the user as an ID token in a standardized, REST-like manner. The client application, or Relying Party, requests information about a user from an authorization server, which is known as an OpenID Connect Provider. After users are authenticated by an OpenID Provider, they can access any Relying Party that is configured for that Provider, without having to manage unique accounts for each application or website.
 
 - link:/docs/ref/general/#/docs/concept/sso-config-saml.html[Security Assertion Markup Language (SAML 2.0)] +
 SAML Web Browser SSO enables web applications to delegate user authentication to a SAML identity provider instead of a configured user registry.
 
 - link:/docs/ref/feature/#spnego-1.0.html[Simple and Protected GSS-API Negotiation Mechanism (SPNEGO)] +
 With SPNEGO SSO, users log in to the Microsoft Domain controller once to access and move between protected resources on Open Liberty servers without getting prompted for additional logins.
 
+= JAAS authentication with Open Liberty
```

Comment:
Not sure about the JAAS section, it feels overly verbose compared with the rest of the page.

Reply:
agreed- see comment above. Can this section get buy with less info? How much detail is relevant at this level of overview? Resources in KC go on for pages detailing LoginModules and CallbackHandlers etc. Here, just introducing the concepts, but is that still overkill in this context?

Reply:
How about something like this:

>Open Liberty uses the Java Authentication and Authorization Service (JAAS) to authenticate users.
>JAAS makes use of named login contexts that are made up of a number of login modules that combine to authenticate a user.
>Open Liberty contains a number of built in login contexts and login modules which are used, but additional modules and contexts can be configured to customize authentication.
>During authentication the JAAS login modules authenticates the users credentials and loads user and group information.
>Depending on the kind of credentials provided the user and group information could be fetched from a UserRegistry, or an SSO token.

Reply:
@NottyCode added this section with a few minor edits